docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
+    * **ConfigBucketName**&mdash;Name of a new S3 bucket that will be created automatically by the stack. This bucket will be used to store certain configuration files associated with your catalog instance. The bucket name must be globally unique, e.g. it cannot point to a bucket that already exists (even if that bucket is in your account).
 
-* **BucketIcon** - This is a URL that points to a square icon that will display as part of your bucket's entry in the dropdown manu.
+      We recommend a bucket name ending in `-config`, to make it more obvious that this bucket is configuration-only.
+    * **DBPassword**&mdash;The auth database password. This doesn't need to be very secure because it will not be publicly accessible. You should only need to access the auth database directly in unusual circumstances.
+    * **DefaultSender**, **MixPanelToken**, **ProductCode**&mdash;These are Quilt-set fields with autogenerated values that can be safely ignored.
+    * **QuiltBucketName**&mdash;Name of the S3 bucket that this catalog instance will be based out of. This should be the same S3 bucket that you configured for access authorization in step 2.
+    * **QuiltWebHost**&mdash;The URL that your catalog will be served out of. Must pattern match the AWS certificate you provide to the `CertificateARN` field. For example, an AWS certificate for `*.foo.com` allows for `t4.foo.com` or `catalog.foo.com` as your `QuiltWebHost`, but not `t4.notfoo.com`.
+    * **RegistryHost**&mdash;The URL that the T4 auth service will be served out of. As with `QuiltWebHost`, this field must pattern match the AWS certificate you provide in `CertificateARN`. It may not be the same value as `QuiltWebHost`.
 
-* **BucketTitle** - Human-friendly title for your bucket to be displayed in the dropdown menu.
+      We recommend appending `-registry` to namespace you set in `QuiltWebHost`. For example, `catalog.foo.com` and `catalog-registry.foo.com`.
+    * **SecretKey**&mdash;Used for session authorization. Provide a random value to this field (e.g. by running `run uuidgen | sha256sum` in the command line).
+    * **SentryDNS** &mdash; Set this field to "`-`".
+    * **Smtphost**, **Smtpassword**, **Smtpusername**&mdash;Log-in information for an SMTP mail server. These fields are necessary because the T4 catalog will need to send (very occassional) service emails on your behalf.
+    * **Users**&mdash;Comma-separated list of ARNs of IAM users and/or IAM roles that will be able to perform searches in the bucket.
 
-* **CertificateArn** - This is the arn for your AWS-managed SSL certificate for your QuiltWebHost domain. Use AWS Certificate Manager to set this up.
+      Note that users with search access can see the entire contents of the bucket. Use with care.
```

Comment:
@akarve I think we should word this differently -- users with search access can't really access the entire bucket, but they can read everything in the index. Is that too subtle of a distinction to matter here?